src/Controller/Action/ListWishlistProductsAction.php

```diff
@@ -102,21 +102,27 @@ public function __invoke(Request $request): Response
         );
     }
 
-    private function handleCartItems(FormInterface $form): bool
+    private function handleCartItems(array $wishlistProductsCommand): void
     {
-        $result = false;
-
-        /** @var AddToCartCommandInterface $command */
-        foreach ($form->getData() as $command) {
-            if (0 < $command->getCartItem()->getQuantity()) {
-                $result = true;
-                $this->orderModifier->addToOrder($command->getCart(), $command->getCartItem());
-                $this->cartManager->persist($command->getCart());
+        foreach ($wishlistProductsCommand as $wishlistProducts) {
+            /** @var AddWishlistProduct $wishlistProduct */
+            foreach ($wishlistProducts as $wishlistProduct) {
+                $addToCartCommand = $wishlistProduct->getCartItem();
+                $cart = $addToCartCommand->getCart();
+                $cartItem = $addToCartCommand->getCartItem();
+
+                if (!$cartItem->getVariant()->isInStock()) {
```

Comment:
Simplify this method, please. It also carries many possibilities and increases complexity: https://en.wikipedia.org/wiki/Cyclomatic_complexity